patches/tModLoader/Terraria/Initializers/AchievementInitializer.cs.patch

```diff
@@ -0,0 +1,10 @@
+--- src/TerrariaNetCore/Terraria/Initializers/AchievementInitializer.cs
++++ src/tModLoader/Terraria/Initializers/AchievementInitializer.cs
+@@ -606,7 +_,6 @@
+ 			Main.Achievements.RegisterAchievementCategory("GLORIOUS_GOLDEN_POLE", category);
+ 			Main.Achievements.RegisterAchievementCategory("TEMPLE_RAIDER", category);
+ 			Main.Achievements.RegisterAchievementCategory("LIKE_A_BOSS", category);
+-			Main.Achievements.Load();
```

Comment:
Comment out instead of removing. ideally using block comments to avoid changing indent

Reply:
As per style guide given above, the block comments start and end should be on separate lines. The indentation of the vanilla code should not change